vmware/ngc/NGC-AdapterManager/src/main/java/org/opensds/vmware/ngc/adapters/opensds/Main.java

```diff
@@ -0,0 +1,20 @@
+package org.opensds.vmware.ngc.adapters.opensds;
```

Comment:
How this file will work?

Reply:
The file was for local testing purpose, pushed by mistake. 
I have removed this file.